.travis.yml

```diff
@@ -3,7 +3,7 @@ cache: ccache
 os: linux
 matrix:
   include:
-    - name: "First commit message adheres to guidelines at <a href=\"https://goo.gl/p2fr5Q\">https://goo.gl/p2fr5Q</a>"
+    - name: "Please follow commit message guidelines: <a href=\"https://goo.gl/p2fr5Q\">link</a>"
```

Comment:
I'm not going to block this, but I'm of the opinion this is less clear than what was there before. The job name shows up in the Travis checks and are basically statements with pass/fail states:
![image](https://user-images.githubusercontent.com/5445507/53585508-f93d3680-3b7d-11e9-8297-c13c15a7ea43.png)

The new name is no longer a statement and instead is a request/instruction. It also now drops the "first commit" which indicated that currently only the first commit is message linted.

Reply:
If you update your PR, adding other commit travis recheck the last commit added

Reply:
No, Travis always runs when the branch the pull request is created from is updated, but the commit message check always checks the first commit (and currently none of the subsequent ones).

Reply:
Oh, yeah, I didn't even notice that this was the job name rather than text displayed on error. I agree the current text is better.

Reply:
> No, Travis always runs when the branch the pull request is created from is updated, but the commit message check always checks the first commit (and currently none of the subsequent ones).

And, for the record, that is a design choice rather than a bug. Subsequent commits are usually squashed into the first commit so that's the one that needs a valid message in most cases.

Reply:
I think it's better to check the last commit, and the previous commits crush them on the last because the last is definitive

Reply:
> I think it's better to check the last commit, and the previous commits crush them on the last because the last is definitive

No. The commit message of the first message is almost always the one to check. Subsequent commits are often/usually fixup commits.

Reply:
(And regardless, this PR doesn't change that behavior anyway.)